refinery/core/search_indexes.py

```diff
@@ -65,15 +65,18 @@ def prepare_submitter(self, object):
         submitters = []
 
         for contact in investigation.contact_set.all():
-            submitters.append(contact.last_name + ", " + contact.first_name)
+            submitters.append(
+                "{}, {}".format(contact.last_name, contact.first_name)
+            )
 
         studies = investigation.study_set.all()
         for study in studies:
             for contact in study.contact_set.all():
                 submitters.append(
-                    contact.last_name + ", " + contact.first_name)
+                    "{}, {}".format(contact.last_name, contact.first_name)
+                )
 
-        return set(submitters)
+        return list(set(submitters))
```

Comment:
This is not redundant. We cast to `set` to get a unique list of `submitters` and back to `list` to fix a search bug. 

Solr kept the `set` in its index like so: `submitters: "set(u'Armstrong, Scott')"`, but
breaks this field on whitespaces.
Due to some prefix searching going on, one could search for `Scott` and `set(u'Armstrong` but not for `Armstrong` alone